mobile.js

```diff
@@ -1,10 +1,101 @@
+
 const hamburgerMenu = document.querySelector('.hamburger-menu');
 const navMenu = document.querySelector('.nav-menu');
+const body = document.querySelector('body');
+const popup = document.querySelector('.button-line');
+const popupbody = document.querySelector('.popup-body');
+const projectWork = document.querySelector('.main-works');
+
 hamburgerMenu.addEventListener('click', () => {
   hamburgerMenu.classList.toggle('active');
   navMenu.classList.toggle('active');
-});
+});popup-body
 document.querySelectorAll('.nav-link').forEach((navLink) => navLink.addEventListener('click', () => {
   hamburgerMenu.classList.remove('active');
   navMenu.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+const displayProjectDetails = ({ LiveLink, SourceLink, description, image, technologies, name })=>{
+  
+    let div = document.createElement('div');
+    div.className='elementpop';
+    div.innerHTML = `
+    <div class=inner-pop-btn>
+    <a class="close-button" id="close-btn" type="button" href="">X</a>
+    <div class="popup-img"><img src="${image}"/></div>
+    </div>
+    <div class="popup-data">
+        <p class="pop-para">${name}</p>
+        <ul class="content-inner-pop">
+            <li class="pop-btn"> <a class="button-pop" type="button" href="">${technologies[0]}</a></li>
+            <li class="pop-btn"> <a class="button-pop" type="button" href="">${technologies[2]}</a></li>
+            <li class="pop-btn"> <a class="button-pop" type="button" href="">${technologies[3]}</a></li>
+        </ul>
+    </div>
+    <p class="pop-desc">${description}</p>
+    <div class=main-works-pop>
+        <a class="button button-line" type="button" href="${LiveLink}">See Live <span><img src="media/github.svg"/></span></a>
+        <a class="button button-line" type="button" href="${SourceLink}">See Source <span><img id="img-pop" src="media/github.svg"/></span></a>  
+    </div>
```

Comment:
Please you're required to follow the provided design as much as possible
- [ ] the icon on the `see live` should be the same as on FIGMA
- [ ] the icon's color should be white as on Figma
- [ ] the `main-works-pop` should be at the right side of the pop-up just below the image